<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1f1e2d;
color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Imágenes/Screenshots/Captura de pantalla 2025-10-10 003256.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableProcesos">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>50</y>
      <width>441</width>
      <height>481</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
    background-color: #26253a;             
    color: #f0f0f0;                        
    gridline-color: #3a3950;               
    selection-background-color: #ff7b00;   
    selection-color: #ffffff;             
    alternate-background-color: #2d2c44;   
    border: none;
    border-radius: 6px;                  
    padding: 2px;
}

QHeaderView::section {
    background-color: #2f2e48;             
    color: #ff9500;                       
    font-weight: bold;
    border: none;
    padding: 8px;
    border-bottom: 2px solid #ff7b00;     
}

QTableCornerButton::section {
    background-color: #2f2e48;
    border: none;
}

/* ScrollBar vertical */
QScrollBar:vertical {
    background: #1f1e2d;
    width: 10px;
    margin: 2px;
    border-radius: 5px;
}

QScrollBar::handle:vertical {
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:0,
        stop:0 #ff7b00,
        stop:1 #ffae42
    );                                    
    border-radius: 5px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:1, y2:0,
        stop:0 #ff9500,
        stop:1 #ff7b00
    );
}

/* Opcional: bordes suaves en hover */
QTableWidget::item:hover {
    background-color: rgba(255, 123, 0, 0.2);  
    border-radius: 4px;
}
</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>PID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NOMBRE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CPU %</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RAM %</string>
     </property>
    </column>
   </widget>
   <widget class="QProgressBar" name="progressBarCPU">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid #ff4fa0;
    border-radius: 12px;
    text-align: center;
    background-color: #1a1a2e;
    color: #ffffff;
    font-weight: bold;
    padding: 2px;
}

QProgressBar::chunk {
    border-radius: 10px;
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #ff4fa0,
        stop: 0.5 #d94fff,
        stop: 1 #ff85d0
    );
    /* brillo suave tipo neón */
    box-shadow: 0px 0px 12px #ff4fa0;
}
QProgressBar {
    border: 2px solid #ff4fa0;
    border-radius: 12px;
    text-align: center;
    background-color: #1a1a2e;
    color: #ffffff;
    font-weight: bold;
    padding: 2px;
}

QProgressBar::chunk {
    border-radius: 10px;
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #ff4fa0,
        stop: 0.5 #d94fff,
        stop: 1 #ff85d0
    );
    /* brillo suave tipo neón */
    box-shadow: 0px 0px 12px #ff4fa0;
}
</string>
    </property>
    <property name="value">
     <number>49</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarMem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid #00ff85;
    border-radius: 12px;
    text-align: center;
    background-color: #1a1a2e;
    color: #ffffff;
    font-weight: bold;
    padding: 2px;
}

QProgressBar::chunk {
    border-radius: 10px;
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #00ff85,
        stop: 0.5 #00d46a,
        stop: 1 #66ffb2
    );
    box-shadow: 0px 0px 12px #00ff85;
}
</string>
    </property>
    <property name="value">
     <number>49</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarDisc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {
    border: 2px solid #3a7dff;
    border-radius: 12px;
    text-align: center;
    background-color: #1a1a2e;
    color: #ffffff;
    font-weight: bold;
    padding: 2px;
}

QProgressBar::chunk {
    border-radius: 10px;
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #3a7dff,
        stop: 0.5 #1f5fff,
        stop: 1 #6aa8ff
    );
    box-shadow: 0px 0px 12px #3a7dff;
}
</string>
    </property>
    <property name="value">
     <number>49</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CPU</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MEMORIA</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DISCO</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonKill">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>540</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>KILL</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphic1" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>361</width>
      <height>161</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Yi Baiti</family>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #26253a; 
color: #ff4fa0;</string>
     </property>
     <property name="text">
      <string>CPU</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>546</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Filtrar&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>546</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Intervalo(ms)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="SpinCantidad">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>540</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SpinIntervalo">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>540</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1000</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="PlotWidget" name="graphic2" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>0</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Yi Baiti</family>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #26253a; 
color: #00ff85;</string>
     </property>
     <property name="text">
      <string>MEMORIA</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="PlotWidget" name="graphic3" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Yi Baiti</family>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #26253a; 
color: #3a7dff;</string>
     </property>
     <property name="text">
      <string>DISCO</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="TotalDisc">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>550</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRefresh">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>560</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>REFRESH</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    padding: 2px 4px; 
}</string>
    </property>
    <property name="text">
     <string>Filtrar Nombre</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEditFiltroNombre">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>13</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid #ff7b00;       
    border-radius: 8px;             
    padding: 6px;                    
    background-color: #1f1e2d;     
    color: #ffffff;               
    selection-background-color: #ff7b00; 
    selection-color: #000000;       
    font-weight: bold;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="UsadoDisc">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>550</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Usado:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LibreDisc">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>550</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Libre:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TotalMem">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="UsadoMem">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Usado:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LibreMem">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Libre:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
